.npm-upgrade.json

```diff
@@ -79,6 +75,14 @@
     "react-dom": {
       "versions": "^18",
       "reason": "Waiting on react"
+    },
+    "jest": {
```

Comment:
Does Dependabot pick up this file?

Reply:
Not at all, but that'd be really cool.

Reply:
Might want to replicate these into the dependabot config so it doesn't try to update these.